### DISCUSSION TITLE
set workspace by „code .“ command in the terminal

### DISCUSSION THREAD

**Discussion by Drei-E3** [thebaselab/codeapp#1132] [Ideas]
When i use VSCode, there is a useful feature, which supports to set current working dictionary or specific dictionary as workspace with „code .“ or „code <specific path>“ command. This is really helpful when i do remote work via ssh

**Comment by bummoblizard** [comment]
Great idea. However this requires installing a command on the server which isn't ideal. I'll need to think how we get around this.

**Comment by bummoblizard** [comment]
As a workaround, you may hold any folder and select open as workspace.
